src/table.js

```diff
@@ -35,7 +35,12 @@ const ChunkTransformer = require('./chunktransformer.js');
 
 // See protos/google/rpc/code.proto
 // (4=DEADLINE_EXCEEDED, 10=ABORTED, 14=UNAVAILABLE)
-const RETRY_STATUS_CODES = new Set([4, 10, 14]);
+const MUTATE_ROWS_RETRYABLE_STATUS_CODES = new Set([4, 10, 14]);
+
+// TODO - DO NOT MERGE - Figure out exactly what codes need to be here. Some combination of
```

Comment:
@sduskis Any input for this?

Reply:
The Java client uses the same statuses for MutateRows and ReadRows.  I'm having a bit of trouble figuring out how the HTTP codes translate... Why can't we just use gRPC status codes?

Reply:
[Because the commonGRPC library "helpfully" converts the grpc code back to it's HTTP equivalent](https://github.com/googleapis/nodejs-common-grpc/blob/master/src/service.js#L570)